### PR TITLE
Upgrade Parent POM

### PR DESCRIPTION
**Pull request by jenniferboedker**
The new parent POM 3.1.0 is integrated, braking dependencies are updated accordingly.

Some minor refactoring of the POM has been done to improve the readability

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,13 +41,93 @@
...
+    </dependency>
+    <dependency>
+      <artifactId>spock-core</artifactId>
+      <groupId>org.spockframework</groupId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <artifactId>data-model-lib</artifactId>
+      <groupId>life.qbic</groupId>
+      <version>1.9.4</version>
```
Comment (sven1103) [inline pom.xml:79]:
```suggestion
      <version>1.11.0</version>
```

**Review by sven1103 — CHANGES_REQUESTED** [review]
One small change, you can directly reference the new data model lib

**Review by sven1103 — APPROVED** [review]
perfect!

File: .travis.yml [hunk 2]
```diff
@@ -22,8 +22,8 @@ branches:
   only:
   - master
   - development
-  - release/*
-  - hotfix/*
+  - release\/.*
+  - hotfix\/.*
```
Comment (sven1103) [inline .travis.yml:26]: I think in travis u need to encapsulate regex with two slashes:
```suggestion
  - /^release\/.*$/
  - /^hotfix\/.*$/
```

Comment (jenniferboedker) [reply .travis.yml:26]: lets try it out

**Review by sven1103 — APPROVED** [review]
lgtm

**Pull request merged**